scripts/build-package.sh

```diff
@@ -222,22 +233,41 @@ checkEnv
 
 [ ! -z "$VERSION" ] || die "Expected a version number as a parameter"
 
+echo -e "Going to build Piwik $VERSION (Major version: $MAJOR_VERSION)"
+
+if [ "$MAJOR_VERSION" == "$CURRENT_LATEST_MAJOR_VERSION" ]
+then
+	echo -e "-> Building a new release for the current latest major version (stable or beta)"
+	BUILDING_LATEST_MAJOR_VERSION_STABLE_OR_BETA=1
+else
+	echo -e "-> Building a new (stable or beta) release for the LONG TERM SUPPORT LTS (not for the current latest major version!) <-"
+	BUILDING_LATEST_MAJOR_VERSION_STABLE_OR_BETA=0
+fi
+
+echo -e "Proceeding..."
+sleep 2
+
 ############################
 echo "Starting build...."
 ############################
 
+
 [ -d "$LOCAL_ARCH" ] || mkdir "$LOCAL_ARCH"
 
 cd $BUILD_DIR
 
-
 if ! [ -d $LOCAL_REPO ]
 then
 	git clone "$URL_REPO" "$LOCAL_REPO"
 fi
 
+# we need to exclude LFS files from the upcoming git clone/git checkout,
+# unfortunately this git config command does not work...
+git config lfs.fetchexclude "tests/"
```

Comment:
this `git config lfs.fetchexclude "tests/"` is not working for some reasons. I'd like the build script to skip all LFS files so the build is faster, but couldn't make it work...